api/catalog.go

```diff
@@ -31,8 +31,11 @@ type CatalogService struct {
 	ServicePort              int
 	ServiceWeights           Weights
 	ServiceEnableTagOverride bool
-	CreateIndex              uint64
-	ModifyIndex              uint64
+	// DEPRECATED (ProxyDestination) - remove the next comment!
```

Comment:
?

Reply:
Heh we decided to use a consistent greppable comment everywhere that contains code that is removable when it's time to kill managed proxies for good. one of those places was this comment which I added earlier in the life of the branch about how we forgot to add them to this API struct in the first place!